target/linux/ramips/dts/WNPR2600G.dts

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "iodata,wnpr2600g", "mediatek,mt7621-soc";
+	model = "I-O DATA WNPR2600G";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "wnpr2600g:green:power";
+			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		notification {
+			label = "wnpr2600g:green:notification";
+			gpios = <&gpio0 12 GPIO_ACTIVE_HIGH>;
+		};
+
+		wlan2g {
+			label = "wnpr2600g:green:wlan2g";
+			gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
+		};
+
+		wlan5g {
+			label = "wnpr2600g:green:wlan5g";
+			gpios = <&gpio0 15 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		auto {
+			label = "auto";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+			linux,input-type = <EV_SW>;
+		};
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x000000 0x030000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x030000 0x010000>;
+				read-only;
+			};
+
+			Factory: partition@40000 {
+				label = "Factory";
```

Comment:
Never seen an upper-case label before. But DTS says it's allowed.
Consider converting (both) to lower-case like all the others.

Reply:
```Factory``` is used in several boards.
```
$ grep "label = \"Factory\"" target/linux/ramips/dts/*
target/linux/ramips/dts/M2M.dts:                                label = "Factory";
target/linux/ramips/dts/MIR3G.dts:                      label = "Factory";
target/linux/ramips/dts/MIR3P.dts:                      label = "Factory";
target/linux/ramips/dts/WN-AC1167GR.dts:                                label = "Factory";
target/linux/ramips/dts/WN-AC733GR3.dts:                                label = "Factory";
target/linux/ramips/dts/WN-AX1167GR.dts:                                label = "Factory";
target/linux/ramips/dts/WN-GX300GR.dts:                         label = "Factory";
target/linux/ramips/dts/WNPR2600G.dts:                          label = "Factory";
target/linux/ramips/dts/WR1201.dts:                             label = "Factory";
```

OK, I'll convert to lower-case.

Reply:
Yes, I've seen it too, but also found lower-case examples. "The others" referred to the other labels in the same DTS in this case.

That's why I explicitly added that DTS allows it: It's just my personal belief that the upper-case is odd, but you are free to choose.

Reply:
I see. That partition is named as ```Factory``` in the OEM firmware, so I've used this name.

OEM log:
```
Creating 8 MTD partitions on "raspi":
0x000000000000-0x000001000000 : "ALL"
0x000000000000-0x000000030000 : "Bootloader"
0x000000030000-0x000000040000 : "Config"
0x000000040000-0x000000050000 : "Factory"
0x000000050000-0x000000df0000 : "Kernel"
0x000000220000-0x000000df0000 : "user"
0x000000df0000-0x000000f80000 : "manufacture"
0x000000f80000-0x000001000000 : "storage"
```